src/bitmessageqt/__init__.py

```diff
@@ -2684,6 +2684,19 @@ def click_NewAddressDialog(self):
     def network_switch(self):
         dontconnect_option = not BMConfigParser().safeGetBoolean(
             'bitmessagesettings', 'dontconnect')
+        reply = QtGui.QMessageBox.question(
+            self, _translate("MainWindow", "Disconnecting")
+            if dontconnect_option else _translate("MainWindow", "Disconnecting"),
```

Comment:
This should be "Connecting" rather than "Disconnecting".

Reply:
oops (: